PyTado/interface/api/hops_tado.py

```diff
@@ -56,15 +56,15 @@ def __init__(
         # set to None until explicitly set
         self._auto_geofencing_supported = None
 
-    def _create_request(self) -> TadoRequest:
+    def _create_x_request(self) -> TadoRequest:
```

Comment:
I just realized, that this method is not useful at all :D, because it changes the way the underlaying Tado class works! So thanks for your change

Reply:
I just realized I forgot to update the return type in the signature to `TadoXRequest`

Reply:
I will add it asap

Reply:
Or better remove this method!